cypress/integration/features/course/deleteCourse.feature

```diff
@@ -0,0 +1,12 @@
+Feature: To delete the course crteated for testing purpose
+
+    Background: User opens Schul-cloud homepage website
+        Given user arrives on the Schul-Cloud homepage
+
+    Scenario Outline: Deleting a new course
+        Given I am logged in as a teacher
+        When '<userRole>' visit the rooms overview
+        Then '<userRole>' should be able to delete the course
+        Examples:
+            | userRole |
+            | teacher  |
```

Comment:
we don need user data table in each feature file since this should be maintained centrally and can be used for any feature as many times.